src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -217,7 +216,16 @@ public function C(){}
 ',
                     ['order' => ['method_public'], 'sort_algorithm' => self::SORT_ALPHA]
                 ),
-            ]
+            ],
+            'Accepts a subset of pre-defined element types, special element groups, and custom patterns.
+
+Element types: `[\''.implode('\', \'', array_keys(self::$typeHierarchy)).'\']`
+
+Special element types: `[\''.implode('\', \'', array_keys(self::$specialTypes)).'\']`
+
+Custom values:
+
+- `method:*`: specify a single method name (e.g. `method:__invoke`) to set the relative order of that specific method'
```

Comment:
```suggestion
- `method:*`: specify a single method name (e.g. `method:__invoke`) to set the explicit order of that specific method'
```

Reply:
Thanks @driftingly for a review, but [I've already pointed this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6360#discussion_r1192831994) 😉

Reply:
Just easier for @bakerkretzmar to include the suggestion 😄